trento/xml/article_sap_trento.xml

```diff
@@ -220,18 +220,16 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
       <title>&t.server; requirements</title>
       <para> Running all the &t.server; components requires a minimum of 2GB of RAM,
         two CPU cores and 64GB of storage.
-        When usng K3s, storage should be provided under <filename>/var/lib/rancher/k3s</filename>,
-        with at least 64 GB.
+        When using K3s, such storage should be provided under <filename>/var/lib/rancher/k3s</filename>.
       </para>
       <para>
-        The &t.server; and the &t.agent;s requires a public SSH key exchange before
-        they can start their communication. To create the public and private SSH keys,
-        log in to the &t.server; as user &trentoadmin; and run the following command:
+       &t.server; needs an SSH key in order to execute configuration checks in the &t.agent;s hosts.
+       You can use an existing SSH key pair or create one by running the following command:
       </para>
       <screen>ssh-keygen -f ~/.ssh/id_rsa -t rsa -b 4096 -N ''</screen>
       <para>
-        <xref linkend="pro-trento-installing-trentoagent"/> describes how to use
-        the public SSH key for the &t.agent;.
+       This command will create two files under the .ssh folder of the home directory of the executing user: 
+       file id_rsa with the private SSH key and file id_rsa.pub with the public SSH key.
```

Comment:
I would like to see these adjusted:

* "folder" is rejected in our styleguide. I'd recommend to avoid that.
* As recommended in our styleguide, I would use simple present tense instead of future tense.
* Could we simplify the structure of the sentence?
   If we use `~/.ssh`, it should become obvious for our target group that we are talking about the user's home directory.  Also the "executing user" should be obvious for those users as well. That would simplify the sentence.
* Some `<filename>` tags are missing.

This would lead to:

```suggestion
       The previous command creates two files in the <filename>~/.ssh</filename> directory:
       file <filename>id_rsa</filename> as private SSH key and file <filename>id_rsa.pub</filename> as public SSH key.
```